css-sizing-4/Overview.bs

```diff
@@ -826,37 +826,24 @@ Extrinsic Size Determination</h2>
 <h3 id="stretch-fit-sizing">
 Stretch-fit Sizing: filling the containing block</h3>
 
-	Stretch-fit sizing tries to set the box’s used size
-	to the length necessary
-	to make its outer size as close to filling the [=containing block=] as possible
-	while still respecting the constraints imposed by min-height/min-width/max-height/max-width.
-
-	Formally, its behavior is the same as specifying an [=automatic size=]
-	together with a [=self-alignment property=] value of ''width/stretch''
-	(in the relevant axis),
-	except that the resulting box,
-	which can end up not exactly fitting its [=alignment container=],
-	can be subsequently aligned by its actual [=self-alignment property=] value.
-
-	Additionally,
-	in [=formatting contexts=] and axes in which the relevant [=self-alignment property=] does not apply
-	(such as the block axis in Block Layout, or the main axis in Flex Layout),
-	in cases where a percentage size in that axis would resolve to a definite value,
-	a [=stretch-fit size=]
-	causes the box to attempt to fill its containing block--
-	behaving as ''100%''
-	but applying the resulting size to its margin box
-	instead of the box indicated by 'box-sizing'.
+	[=Stretch-fit sizing=] indicates the size necessary to make the box’s [=outer size=]
+	fill the [=containing block=].
```

Comment:
What if the box is a block-level that establishes an independent formatting context, and there are floats to avoid?
`justify-self` aligns in the float-shrunken containing block.